docs/src/root.tsx

```diff
@@ -22,7 +22,7 @@ export default function Root() {
             <Head>
                 <Title>MapLibre Style Spec</Title>
                 <Meta charset="utf-8" />
-                <Meta name="viewport" content="width=device-width, initial-scale=1" />
+                <Meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1.0, user-scalable=no, viewport-fit=cover" />
```

Comment:
> Warning: Usage of user-scalable=no can cause accessibility issues to users with visual impairments such as low vision. [WCAG](https://developer.mozilla.org/en-US/docs/Web/Accessibility/Understanding_WCAG/Perceivable#guideline_1.4_make_it_easier_for_users_to_see_and_hear_content_including_separating_foreground_from_background) requires a minimum of 2× scaling; however, the best practice is to enable a 5× zoom.

https://developer.mozilla.org/en-US/docs/Web/HTML/Viewport_meta_tag#viewport_basics

This is not ideal for accessibility.

Reply:
Please do, yes, this is the solution I'm familiar with...